x-hooks/use-bounty.tsx

```diff
@@ -117,7 +127,7 @@ export function useBounty() {
     dispatch(changeSpinners.update({bountyChain: true}))
 
     state.Service.active.getBounty(state.currentBounty.data.contractId)
-      .then(bounty => {
+      .then(async bounty => {
```

Comment:
no asyncs inside a non-async function, use then callback or affect state change so erc20data is loaded on its own side-effect